frontend/src/components/GameController.js

```diff
@@ -26,7 +26,12 @@ const samNodeIcon = L.icon({
 
 let userMarker = L.marker([27.380583, 33.631839], { icon: userIcon });
 
-export default function GameController({ points, setIsLoading, useMap }) {
+export default function GameController({
```

Comment:
This file really big.. I think you can move some functions to a service file. You have a lot of `let`s that should be `const`. Only use `let` for values that do change!

Reply:
I put that on my bucketlist! Thanks for the input! :)